bin/test_piotr.py

```diff
@@ -0,0 +1,8 @@
+import requests
+f = open('../test/queries/english_00.txt', 'r')
```

Comment:
@sudheesh001 I have submitted a PR with exactly the kind of functionality you are talking about :+1: